.github/templates/aarch64_binary_build_workflow.yml.j2

```diff
@@ -0,0 +1,77 @@
+{% import 'common.yml.j2' as common %}
```

Comment:
Hmm, can you please elaborate, why new template is needed rather than refactor existing one?

Reply:
Following macos_binary_build_workflow.yml.j2 as instances, images, and called workflows are different and there is no need for CUDA or ROCM. This lessens the complexity of linux_binary_build_workflow.yml.j2 which has a lot happening.

Reply:
Hmm, if they are not needed, why do you pass `GPU_ARCH_VERSION` parameters to the workflow?

Reply:
As of [faf9383](https://github.com/pytorch/pytorch/pull/104109/commits/faf9383d4a8929c0870d5d903ce7353e539c3818) there is only one template.